embedded-couchbase/pom.xml

```diff
@@ -14,7 +14,7 @@
     <artifactId>embedded-couchbase</artifactId>
 
     <properties>
-        <couchbase-java-client-3.x.version>3.4.7</couchbase-java-client-3.x.version>
+        <couchbase-java-client-3.x.version>3.4.6</couchbase-java-client-3.x.version>
```

Comment:
The CI build is red, this is to try to fix it.

Reply:
this downgrage wasn't helpful. reverted.
_(locally on my machine couchbase module build is green)_